person.rb

```diff
@@ -21,6 +22,10 @@ def correct_name
     @name
   end
 
+  def add_rentals(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- Hey, let's improve the `add_rentals` since, the person could also be the rental and this same person can have many books, and many books could belong to them and they can rent as many books as possible, so keeping track of the date and book the person has rented and including the person itself below:

```ruby 

 def add_rental(book, date)
    Rental.new(date, self, book)
  end
```

Reply:
Thank you. Changes implemented